lib/editing/editor.dart

```diff
@@ -46,6 +46,9 @@ abstract class Editor {
    */
   bool get completionActive;
 
+  bool get completionAutoInvoked;
```

Comment:
You can use a field here (`bool completionAutoInvoked`), which is equivalent to a setter/getter pair. Subclasses can implement it however they want.